### PR TITLE
Task/mm5 5146 misc spelling mistakes

### PR DESCRIPTION
**Pull request by Poiar**
Dictionaries that agree that "folder" in Danish isn't "folder"
[sproget.dk](https://sproget.dk/lookup?SearchableText=folder)
[da.bab.la](https://da.bab.la/ordbog/engelsk-dansk/folder)
[dictzone.com](https://dictzone.com/engelsk-dansk-ordbog/folder)
[ordnet.dk (Den Danske Ordbog)](https://ordnet.dk/ddo/ordbog?query=folder)
[ordbogen.com](https://www.ordbogen.com/da/mobile/article?query=folder&dictionaryId=ordbogen-daen)

ID'et not IDet
https://sproget.dk/lookup?SearchableText=id

### CODE REVIEW

File: DC/5.6.0/languages.dcl [hunk 1]
```diff
@@ -5303,7 +5303,7 @@ resource language_label lbl_assetview_admin_user_mouseover_firstname_danish {
 
 resource language_label lbl_admin_destinationmanager_ftpfoldermustexists_swedish {
     constant = 'LBL_ADMIN_DESTINATIONMANAGER_FTPFOLDERMUSTEXISTS'
-    label = 'FTP folder hase to exists'
+    label = 'FTP folder has to exist'
```
[the remark this answers is not in the document]
Comment (Poiar) [reply DC/5.6.0/languages.dcl:5306]: It's not English either :) If you'd rather that I don't touch other languages where they have an English placeholder, then I'll refrain  👍

File: DC/5.6.0/languages.dcl [hunk 2]
```diff
@@ -18509,7 +18509,7 @@ resource language_label lbl_fjs_meditor_confirm_addall_swedish {
 
 resource language_label lbl_fjs_cm_searches_populate_tooltip_danish {
     constant = 'LBL_FJS_CM_SEARCHES_POPULATE_TOOLTIP'
-    label = 'Generer søge cache'
+    label = 'Generer søge-cache'
```
[the remark this answers is not in the document]
Comment (Poiar) [reply DC/5.6.0/languages.dcl:18512]: Generer = Bydeform/imperativ - aka. kommando/command (Åbn, kør, løb, hop)
Genererer = Nutid/present tense (Åbner, kører, løber, hopper)

This is meant a command, right?

**Comment by madjufra** [comment]
Mathias, seems like we have forgotten this.

When you are fixing comments on PRs, remember to re-request feedback when done, otherwise we dont get a notification to revisit it.

I am closing this one for now

**Comment by Poiar** [comment]
Hey Magnus,

I knew of its existence - but I made it before I was really comfortable with GitHub, and it was way too big. It should have been a lot of smaller PRs. I know now :)

I agree with this being closed. Now I don't have to think about it anymore. Thanks!

**Pull request closed without merging**